src/sip.c

```diff
@@ -626,18 +647,20 @@ sip_parse_msg_media(sip_msg_t *msg, const u_char *payload)
         }
 
         // Check if we have attribute format RTCP port
-        if (!strncmp(line, "a=rtcp:", 7)) {
-            if (media && sscanf(line, "a=rtcp:%hu", &dst.port)) {
-                // Create early RTCP stream
-                if (!rtp_find_call_stream(call, src, dst)) {
-                    call_add_stream(call, stream_create(media, dst, PACKET_RTCP));
-                }
-            }
+        if (!strncmp(line, "a=rtcp:", 7) && rtcp_stream) {
+            sscanf(line, "a=rtcp:%hu", &rtcp_stream->dst.port);
         }
 
 
     }
+
+    // Add streams from last 'm=' line to the call
+    ADD_STREAM(rtp_stream);
+    ADD_STREAM(rtcp_stream);
```

Comment:
When call have multiple rtp streams configured in SDP we will be only adding the last one to the call. I'm commenting just reading the code, most probably there is something I missing here :(